pkg/traversal/worker.go

```diff
@@ -72,6 +81,36 @@ func (w *worker) run() {
 			continue
 		}
 
+		isKubernetesResource := true
+		kubeResource, err := kube.ReadKubernetesResourceFromPath(wf.f.AbsPath())
+		if err != nil {
+			if err == kube.NoKubernetesResourceError {
+				isKubernetesResource = false
+			} else {
+				w.errorCh <- &fileProcessingError{
+					path:  wf.f.Path(),
+					cause: err,
+				}
```

Comment:
`continue` after sending the error down the error channel?

Reply:
great catch, fixing